### PR TITLE
Update abs_Min.py

### PR DESCRIPTION
**Pull request by SandersLin**
from Maths.abs import absVal does not work.

### CODE REVIEW

**Comment by cclauss** [comment]
Can you please rename the file to be all lowercase?

**Comment by AnupKumarPanwar** [comment]
@SandersLin Please resolve the conflicts

File: maths/abs_min.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from Maths.abs import absVal
+from abs import absVal
```
Comment (AnupKumarPanwar) [inline maths/abs_min.py]: absVal doesn't exist in abs.py

Comment (SandersLin) [reply maths/abs_min.py]: This should resolve the conflicts. Thanks

**Pull request merged**